unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarShape.cs

```diff
@@ -157,17 +162,13 @@ public override IEnumerator ApplyChanges(BaseModel newModel)
 
             yield return null; //NOTE(Brian): just in case we have a Object.Destroy waiting to be resolved.
 
-            // To deal with the cases in which the entity transform was configured before the AvatarShape
-            if (!initializedPosition && scene.componentsManagerLegacy.HasComponent(entity, CLASS_ID_COMPONENT.TRANSFORM))
-            {
-                initializedPosition = true;
-                OnEntityTransformChanged(entity.gameObject.transform.localPosition,
-                    entity.gameObject.transform.localRotation, true);
-            }
+            yield return new UnityEngine.WaitUntil(PositionCorrectlyInitialized);
+            
+            Debug.Log("DEBUG: " + entity.gameObject.transform.localPosition);
```

Comment:
Please remove the log

Reply:
Thanks Misha! Sorry for the silly mistake